build/main-branch-sync/upstream-release.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+set -e
+
+SCRIPT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [[ -z "${NEW_RELEASE}" ]]; then
+  echo "error: Export NEW_RELEASE with the new vX.Y.Z version before running this script."
+  exit 1
+fi
+
+echo "=== Tagging upstream repos with ${NEW_RELEASE}"
+REPOS="$(${SCRIPT_PATH}/fetch-repo-list.sh | grep -v 'collection\|generator\|nucleus')"
```

Comment:
I went with a deny list here, but this could be an allow list (not that the repos here change all that much...)